scripts/uniswap-split-route-callee/index.js

```diff
@@ -0,0 +1,96 @@
+import path from 'path';
+import url from 'url';
+import * as dotenv from 'dotenv';
+import { spawn } from 'child_process';
+import { AlphaRouter, ChainId, SwapType } from '@uniswap/smart-order-router';
+import { CurrencyAmount, Percent, Token, TradeType } from '@uniswap/sdk-core';
+import { ethers } from 'ethers';
+
```

Comment:
Can we try and make this script as similar as possible to it's 1inch counterpart?
For example the 1inch has 
Ideally the scripts would be almost identical in a diff except some well contained parts.

Would be good if things like this are the same when diffing - order of imports, structure and name of functions, rpc related definitions/assignments, link input amounts...

Reply:
So far i have adjusted the import order and the usage of the env variable.

For other adjustments (if they are necessary) i would have the following questions:

  1. Are you implying that i should also adjust the 1inch script to achieve the maximal similarity?
  2. By similarities "structure of functions" do you mean "as many similar lines as possible"? something else?
  3. Since from my perspective the processes of setting up the tests via scripts are quite different, i would also not expect the scripts to be similar. So in order to make a meaningful adjustments, i would appreciate if you could select 1 particular instance of something that should be adjusted, describe (on very general level) what the adjustment would look like and what the adjustment improves particularly (or 1 main point). Then i will take this with me and propagate the approach on the whole file.

Reply:
1. If you determine that something in the 1inch script needs to change due to the way it's used in the Uniswap script then yes. If it's just a style issue or a non-mandatory improvement then at this point I would just do it like the 1Inch script (and possibly improve both in a separate PR).

2. I mean that when doing a visual diff it will be very clear what's different between 1Inch and Uniswap router. See (3) for an example. Eventually, the scripts do (or should do) the same thing, just using different platforms.

3. An example is the new script has a `getProvider` function while the 1Inch script does not. I understand you need the provider in more places in the new script so in this case I'd advise aligning the 1Inch script (and making sure to re-test it ofc).
Another example is the new `getLinkAmountToSwap` function, we didn't have that before. It might be currently needed since you differentiate the profit vs profit cases according to the input amount, but it should actually be done according to when the take is done during the auction (see below). If we use a fix as in the 1Inch then we can make that part similar as well.

Reply:
- reshuffled the function structure a bit
- the amount to swap is now hardcoded.